.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+_Steps to reproduce the behavior:_
+1. Go to '...'
+2. Click on '....'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System info**
```

Comment:
Are those necessary info for our application?

Reply:
Nope. I based this on another project 😆 I'll remove it now.

Reply:
Nope, I based it on another project 😆 I'll remove it now.